mirrord/layer/tests/configs/outgoing_filter.json

```diff
@@ -0,0 +1,11 @@
+{
+    "feature": {
+        "network": {
+            "outgoing": {
+                "filter": {
+                    "remote": ["tcp://1.1.1.0/24:1111", "2.2.2.2:2222"]
```

Comment:
Why does the test pass if in the list of peers there is also `3.3.3.3:3333`, but it's not in the filter?

Reply:
Makes no sense, I'm investigating.

Running it as an app, mirrord behaves as expected, where if it's not in the `remote` list, it doesn't go through the agent.